kanban-board/src/components/Issue/Issue.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Card,
+  CardContent,
+  Typography,
+  CardHeader,
+  Avatar,
+} from "@material-ui/core";
+import { Component, ReactNode } from "react";
+import { dragBegin } from "../../common/actions";
+import { Status } from "../../common/constants";
+import { IIssue } from "../../common/types";
+import { AppDispatch } from "../../store/store";
+import './Issue.css';
+
+interface IIssueProps {
+  issue: IIssue,
+  id: number,
+  dispatch: AppDispatch,
+}
+
+class Issue extends Component<IIssueProps> {
+  handleDragStart(index: number) {
+    this.props.dispatch(dragBegin(index));
+  }
+
+  render(): ReactNode {
```

Comment:
An abstract component structure we use is:
1. component hooks 
2. render 
3. other methods

Reply:
done